homeassistant/components/ccm15/config_flow.py

```diff
@@ -0,0 +1,74 @@
+"""Config flow for Midea ccm15 AC Controller integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from ccm15 import CCM15Device
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+import homeassistant.helpers.config_validation as cv
+
+from .const import DEFAULT_TIMEOUT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Optional(CONF_PORT, default=80): cv.port,
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> dict[str, Any]:
```

Comment:
This function is now quite small, you can inline this now, so you can also remove the `CannotConnect` exception